jupyter/scripts/incremental_learning/experiment_driver.py

```diff
@@ -128,42 +132,109 @@ def read_dataset(_run, dataset_name):
 
 
 @ex.capture
-def transform_dataset(dataset: pd.DataFrame, transform_name: str, transform_parameters, _seed: int) -> pd.DataFrame:
+def transform_dataset(dataset: pd.DataFrame,
```

Comment:
Not a big deal, but it would be nice to add types for the `transform_parameters` variable. It appears to be a dict based on later in the code. Given that the rest of the function params have type info, why doesn't this one?

Reply:
I agree in principle, but I inherited this. I think @valeriy42 said he ran into issues defining the type of this with sacred. Since he's away at the moment and it isn't really part of this PR (I just added some extra parameters) I think I'll leave it for the time being and revisit when I have everything working end-to-end.